x-pack/plugins/code/public/components/admin_page/project_tab.tsx

```diff
@@ -27,21 +28,39 @@ const NewProjectButton = styled(EuiButton)`
   margin-top: 1.5rem;
 `;
 
-// TODO(qianliang)
+enum SortOptionsValue {
+  alphabetical_asc = 'alphabetical_asc',
+  alphabetical_desc = 'alphabetical_desc',
+  updated_asc = 'updated_asc',
+  updated_desc = 'updated_desc',
+  recently_added = 'recently_added',
+}
+
+const sortFunctions: { [k: string]: (a: Repository, b: Repository) => number } = {
+  [SortOptionsValue.alphabetical_asc]: (a: Repository, b: Repository) =>
+    a.name! > b.name! ? 1 : -1,
+  [SortOptionsValue.alphabetical_desc]: (a: Repository, b: Repository) =>
+    a.name! < b.name! ? 1 : -1,
+  // [SortOptionsValue.updated_asc]: (a: Repository, b: Repository) => a > b,
```

Comment:
@mw-ding I don't think we have the knowledge for sorting by last update time and adding time.

Reply:
Yea, let it do nothing for now. We will come back to all places with recent update timestamp later.